### PR TITLE
Moves where the pulp-openapi-generator is cloned

### PR DESCRIPTION
**Pull request by dkliban**
[noissue]

### CODE REVIEW

File: templates/travis/.travis/before_install.sh.j2 [hunk 1]
```diff
@@ -26,12 +26,14 @@ then
...
   export PULP_BINDINGS_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-openapi-generator\/pull\/(\d+)' | awk -F'/' '{print $7}')
   export PULP_OPERATOR_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-operator\/pull\/(\d+)' | awk -F'/' '{print $7}')
+  export PULP_BINDINGS_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-openapi-generator\/pull\/(\d+)' | awk -F'/' '{print $7}')
 else
   export PULP_PR_NUMBER=
   export PULP_SMASH_PR_NUMBER=
   export PULP_ROLES_PR_NUMBER=
   export PULP_BINDINGS_PR_NUMBER=
   export PULP_OPERATOR_PR_NUMBER=
+  export PULP_BINDINGS_PR_NUMBER=
```
Comment (mikedep333) [inline templates/travis/.travis/before_install.sh.j2]: Already exists 2 lines above.

File: templates/travis/.travis/before_install.sh.j2 [hunk 2]
```diff
@@ -26,12 +26,14 @@ then
   export PULP_ROLES_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/ansible-pulp\/pull\/(\d+)' | awk -F'/' '{print $7}')
   export PULP_BINDINGS_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-openapi-generator\/pull\/(\d+)' | awk -F'/' '{print $7}')
   export PULP_OPERATOR_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-operator\/pull\/(\d+)' | awk -F'/' '{print $7}')
+  export PULP_BINDINGS_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-openapi-generator\/pull\/(\d+)' | awk -F'/' '{print $7}')
```
Comment (mikedep333) [inline templates/travis/.travis/before_install.sh.j2]: Already exists 2 lines above.

File: templates/travis/.travis/before_install.sh.j2 (hunk 1 above)
Comment (dkliban) [reply templates/travis/.travis/before_install.sh.j2]: updated

File: templates/travis/.travis/before_install.sh.j2 (hunk 2 above)
Comment (dkliban) [reply templates/travis/.travis/before_install.sh.j2]: updated

**Pull request merged**